source/Main.hx

```diff
@@ -18,8 +18,8 @@ class Main extends Sprite
 	var initialState:Class<FlxState> = TitleState; // The FlxState the game starts with.
 	var zoom:Float = -1; // If -1, zoom is automatically calculated to fit the window dimensions.
 	var framerate:Int = 60; // How many frames per second the game should run at.
-	var skipSplash:Bool = true; // Whether to skip the flixel splash screen that appears in release mode.
-	var startFullscreen:Bool = false; // Whether to start the game in fullscreen on desktop targets
+	var skipSplash:Bool = false; // Whether to skip the flixel splash screen that appears in release mode.
+	var startFullscreen:Bool = true; // Whether to start the game in fullscreen on desktop targets
```

Comment:
wtf this forces fullscreen on startup you should be banned and murdered (irl)

Reply:
imagine forcing fullscreen on startup + ratio + 152.141.239.74

Reply:
574.851.152.68

Reply:
alright hold on im ognna change it